zboxcore/sdk/allocation.go

```diff
@@ -222,15 +222,15 @@ func (a *Allocation) InitAllocation() {
 }
 
 func (a *Allocation) isInitialized() bool {
-	return a.initialized && sdkInitialized
+	return a.initialized && SdkInitialized
```

Comment:
when we run `InitStorageSDK`, normally `sdkInitialized` is set to true. However, in our WASM version, I can't pass this test (cfm. https://github.com/0chain/gosdk/blob/rizary-wallet-zboxsdk/wasm/sdk_test.go#L160) because the `InitStorageSDK` doesn't set the `sdkInitialized` to true. Whereas, the `blockchain` is changed, and other test pass, the `TestReadPool` fail because it's think that sdk is not initialized.

In our other test, like `InitZCNSDK`, it works well.

Reply:
you need more `setup` action on your test. so please don't change anything without any bugfixs , just get your test passed. It may break somethings